geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala

```diff
@@ -0,0 +1,112 @@
+package org.locationtech.geomesa.blob.core
+
+import java.io.File
+
+import com.google.common.io.{ByteStreams, Files}
+import org.apache.accumulo.core.client.admin.TimeType
+import org.apache.accumulo.core.client.{Scanner, TableExistsException}
+import org.apache.accumulo.core.data.{Key, Mutation, Range, Value}
+import org.apache.accumulo.core.security.Authorizations
+import org.apache.hadoop.io.Text
+import org.geotools.data.Query
+import org.geotools.data.collection.ListFeatureCollection
+import org.geotools.data.simple.SimpleFeatureStore
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, _}
+import org.locationtech.geomesa.accumulo.util.{GeoMesaBatchWriterConfig, SelfClosingIterator}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore._
+import org.locationtech.geomesa.blob.core.handlers.BlobStoreFileHandler
+import org.locationtech.geomesa.utils.geotools.Conversions._
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.opengis.feature.simple.SimpleFeatureType
+import org.opengis.filter.Filter
+
+import scala.collection.JavaConversions._
+
+class AccumuloBlobStore(ds: AccumuloDataStore) {
+
+  private val connector = ds.connector
+  private val tableOps = connector.tableOperations()
+
+  val blobTableName = s"${ds.catalogTable}_blob"
+
+  ensureTableExists(blobTableName)
```

Comment:
prefer the accumulo interop method:
https://github.com/locationtech/geomesa/blob/master/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/AccumuloVersion.scala#L61